common/src/main/java/us/ilite/common/lib/control/PIDController.java

```diff
@@ -0,0 +1,331 @@
+package us.ilite.common.lib.control;
+/**
+ * This class implements a PID Control Loop.
+ * 
+ * Does all computation synchronously (i.e. the calculate() function must be called by the user from his own thread)
+ */
```

Comment:
Generally speaking, we need a way to log the results of this class in *numerical* form rather than string form.

Reply:
Sorry Jesse can you explain what you mean a little